plugins/wallabag.koplugin/main.lua

```diff
@@ -1042,6 +1059,11 @@ function Wallabag:getLastPercent()
     end
 end
 
+function Wallabag:addToDownloadQueue(articleUrl)
```

Comment:
Coding style, articleUrl would be a function.
```suggestion
function Wallabag:addToDownloadQueue(article_url)
```